Gem/Code/Source/Automation/PrecommitWizardSettings.h

```diff
@@ -0,0 +1,98 @@
+
+        struct ImageDifferenceLevel
+        {
+            enum Levels
+            {
+                NoDifference = 0,
+                LowDifference = 1,
+                ModerateDifference = 2,
+                HighDifference = 3,
+
+                NumDifferenceLevels = 4
+            };
+        };
+        static constexpr const char* ManualInspectionDifferenceLevels[] = {
+            "No Difference",
+            "Low Difference",
+            "Moderate Difference",
+            "High Difference"
+        };
+        static constexpr const char* ManualInspectionOptions[] = {
+            "I don't see any difference",
+            "I see a benign difference",
+            "I see a difference that's *probably* benign",
+            "This looks like a problem"
+        };
+
+        // currently set to track the ScriptReport index and the ScreenshotTestInfo index.
+        using ReportIndex = AZStd::pair<size_t, size_t>;
+
+        // This function does the following:
+        // 1. Collect passing screenshot tests and sorts them by decreasing diff score.
+        // 2. Collect failed screenshot tests and sorts them by decreasing diff score. 
+        void ProcessScriptReports(const AZStd::vector<ScriptReporter::ScriptReport>& scriptReports)
+        {
+            m_reportsOrderedByThresholdToInspect.clear();
+            m_failedReports.clear();
+
+            for (size_t i = 0; i < scriptReports.size(); ++i)
+            {
+                const AZStd::vector<ScriptReporter::ScreenshotTestInfo>& screenshotTestInfos = scriptReports[i].m_screenshotTests;
+                for (size_t j = 0; j < screenshotTestInfos.size(); ++j)
+                {
+                    // Collect and sort reports that passed by threshold. This will be used to detect false negatives
+                    // e.g. a test is reported to pass by being below the threshold when in fact it's simply because the threshold is too
+                    // high
+                    if (screenshotTestInfos[j].m_officialComparisonResult.m_resultCode == ScriptReporter::ImageComparisonResult::ResultCode::Pass)
+                    {
+                        m_reportsOrderedByThresholdToInspect.insert(AZStd::pair<float, ReportIndex>(
+                            screenshotTestInfos[j].m_officialComparisonResult.m_finalDiffScore,
+                            ReportIndex{ i, j }));
+                    }
+                    else
+                    {
+                        m_failedReports.insert(AZStd::pair<float, ReportIndex>(
+                            screenshotTestInfos[j].m_officialComparisonResult.m_finalDiffScore,
+                            ReportIndex{ i, j }));
+                    }
+                }
+            }
+
+            m_reportIterator = m_reportsOrderedByThresholdToInspect.begin();
+        }
+
+        int m_inspectionSelection = DefaultInspectionSelection;
+        Stage m_stage = Stage::Intro;
+        AZStd::string m_exportedPngPath = "";
```

Comment:
You do not need to assign the empty string here. It is an empty string by default.